### PR TITLE
Fix variable naming and extraneous null characters in string

### PR DESCRIPTION
**Pull request by tcwan**
1. brick_name_str not referenced correctly in the previous fix.
2. brick_name_str had extra null characters in the unicode string that needs to be removed, otherwise subsequent strings are not displayed.
3. Misc source file whites pace cleanups

### CODE REVIEW

File: scripts/nxtfilemgr [hunk 1]
```diff
@@ -284,12 +285,11 @@ class NXT_FileMgr(Gtk.Window):
             if prot_version == PROTOCOL_VER:
                 brick_name_b, brick_hostid, brick_signal_strength, brick_user_flash = self.brick.get_device_info()
                 connection_type = str(self.brick.sock)
-                brick_name_str = brick_name_b.decode('windows-1252')
-                print("brick_name_b = {0}, brick_name_str = {1}".format(brick_name_b, brick_name_str))
-
+                brick_name_str = brick_name_b.decode('windows-1252').strip('\00')
```
Comment (dlech) [inline scripts/nxtfilemgr]: I think that `brick.get_device_info()` should return the decoded and stripped value rather than all callers having to do this themselves.

As I recall, the last time I was testing this, it was in fact doing exactly that (tested on Linux and Mac), so I'm not sure why this change would be needed. (I guess I'm going to have to just try it again if I really what to know).

Comment (tcwan) [reply scripts/nxtfilemgr]: Hmm. I'm using nxt-python 2.2.2 which is not the latest master version.
Maybe that is the problem I'm facing, and we need to have a new release version to test with.

Comment (dlech) [reply scripts/nxtfilemgr]: use the `PYTHONPATH` environment variable

Comment (dlech) [reply scripts/nxtfilemgr]: > we need to have a new release version to test with.

Yes, [I made this request recently as well](https://github.com/Eelviny/nxt-python/issues/50#issuecomment-385192950)

Comment (dlech) [reply scripts/nxtfilemgr]: according to the readme of this project, no, it will not be python2 compatible

Comment (tcwan) [reply scripts/nxtfilemgr]: I see. So in that case, I'll still need to maintain my own version of `nxtfilemgr` for 2.2.2.

**Comment by tcwan** [comment]
I guess I'll close this pull request since it is not python3 compatible.
Ongoing work on `nxtfilemgr` should track [nxt_filer enhancement](https://github.com/Eelviny/nxt-python/pull/146)
